### ISSUE TITLE
Layers in model

### ISSUE THREAD

**Issue by tahuy** [satyenrajpal/Concrete-Crack-Detection#10]
Hi satyenrajpal,
As I saw in graph from paper of Young-Jin Cha and Wooram Choi, you have 4 convolution layers, 2 max pool layer, 1 relu layer and 1 softmax. But in your code, you have 3 convolution layers, 2 max pool layer, 1 flatten layer, 1 relu layer and 2 fully connect layers (in fully connect layer you have relu also).

I am unclear about that, so when I you my dataset with 256*256, the accuracy is low although I change the filter and stride size as paper. It make me confuse.

Can you explain for me?
My English is not good, so if it is hard to understand, I will explain more.
Thank,
Huy

**Comment by satyenrajpal** [comment]
Hi Huy, the architecture change was mainly because of different image size inputs. Regarding accuracy, how is the training accuracy?

**Comment by tahuy** [comment]
I still keep your model. After build model, I detect crack on complex picture / video frame (included crack + tree + house + people ...) and the result is not adapt to my expectation. But it good for simple image. Do you have any idea for this issue?

**Comment by satyenrajpal** [comment]
I think this is due to the training data consisting only of cracks in concrete. It does not have images pertaining to the cases that you mentioned (tree+house+people). 
You would need new training data in order for it to work.

**Comment by satyenrajpal** [comment]
Closing this issue. Feel free to re-open
